### DISCUSSION TITLE
Config entry for output directory?

### DISCUSSION THREAD

**Discussion by chrisesler** [comfyanonymous/ComfyUI#381] [General]
I have my outputs from A1111 go to a larger, different drive than the drive the UI is on.

is there a config option for ComfyUI to send outputs to different directory? I'd rather not have to make a symlink.

**Comment by offroadguy56** [comment]
I too wish to change the output directory to a networked drive over SMB.

**Comment by comfyanonymous** [comment]
You can use this command line argument: --output-directory

**Reply by offroadguy56** [reply]
> You can use this command line argument: --output-directory

Neat. Is there any documentation listing the rest of the command line arguments? I'm blind and can't seem to find it on the main GitHub page.

**Reply by comfyanonymous** [reply]
If you use: --help
It will list them.

**Reply by offroadguy56** [reply]
> If you use: --help It will list them.

Nice, thanks
